test/gtest/ucs/test_arbiter.cc

```diff
@@ -890,31 +892,47 @@ test_arbiter_random_resched::dispatch(ucs_arbiter_group_t *_group,
 
     ++m_num_dispatch;
 
+    /* Test ucs_arbiter_group_num_elems() */
+    EXPECT_EQ(group->num_elems, ucs_arbiter_group_num_elems(&group->super));
+
+    /* We should be able to reschedule this group to another place */
+    EXPECT_FALSE(ucs_arbiter_group_is_scheduled(&group->super));
+
     /* Test ucs_arbiter_elem_is_only() */
     if (group->num_elems == 1) {
         ++m_num_only;
         EXPECT_TRUE(ucs_arbiter_elem_is_only(&group->super, elem));
     }
 
+    /* Randomly add few more elements to same group */
+    while ((ucs::rand() % 4) == 0) {
+        add_new_elem(group);
+        if (ucs::rand() % 2) {
+            ucs_arbiter_group_schedule(&m_arb, &group->super);
+        }
+    }
+
     if (ucs::rand() % 2) {
```

Comment:
`((ucs::rand() % 2) == 0)` to be consistent with `if` statement above

Reply:
fixed